config/locales/models.es.yml

```
@@ -155,7 +155,8 @@ es:
         data: 'Datos'
         comments: 'Comentarios'
         tags: 'Etiquetas'
-        created_at: 'Fecha de caso'
+        created_at: 'Creación'
+        updated_at: 'Actualización'
```

```suggestion
        created_at: 'Fecha de caso'
```
No había visto que estaba "sobre escrita", dejaría la original del `created_at`, del `updated_at` no hace falta porque está la "global". Como es lo único lo meto como suggestions y lo megeo.